Cargo.toml

```diff
@@ -39,6 +39,7 @@ thiserror = "1"
 bytes = "1"
 byteorder = "1"
 datafusion = "43"
+datafusion-expr = "43"
```

Comment:
https://docs.rs/datafusion/latest/datafusion/logical_expr/index.html

datafusion already re-export this.